sos/plugins/kernel.py

```diff
@@ -81,7 +82,10 @@ def setup(self):
             "/proc/timer*",
             "/proc/lock*",
             "/proc/misc",
-            "/var/log/dmesg"
+            "/var/log/dmesg",
+            numa_path + "node*/meminfo",
```

Comment:
Ok, I'll move it to numa and will try to fix the above issues.
Feel free to reject this request I'll create new one.